.github/workflows/README.md

```diff
@@ -0,0 +1,60 @@
+# GitHub Workflows
```

Comment:
Instead of duplicating this whole file, let's keep this file in _either_ `reboot-dev/respect` OR here in `3rdparty/dev-tools`. Since most of this documentation is aimed at Reboot employees, let's keep it in `reboot-dev/respect`, and let's have this `README.md` in `3rdparty/dev-tools` just have a link to that `README.md` in `reboot-dev/respect`.